### PR TITLE
Add the accommodation and additional endpoints from the mapper

### PR DESCRIPTION
**Pull request by oleg-n-k**
Add the accommodation and additional endpoints from the mapper

### CODE REVIEW

File: Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs [hunk 1]
```diff
@@ -40,43 +41,118 @@ public MapperManagementClient(IHttpClientFactory clientFactory, ILogger<MapperMa
...
+
+        private async Task<Result<TResponse, ProblemDetails>> Send<TResponse>(HttpRequestMessage requestMessage,
+            string languageCode = LocalizationHelper.DefaultLanguageCode, CancellationToken cancellationToken = default)
         {
-            var client = _clientFactory.CreateClient(HttpClientNames.MapperManagement);
             try
             {
-                using var response = await client.PostAsync(requestUri, content, cancellationToken);
+                _httpClient.DefaultRequestHeaders.Add("Accept-Language", languageCode);
+                var response = await _httpClient.SendAsync(requestMessage, cancellationToken);
```
Comment (DFSko) [inline Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs:123]: HttpResponseMessage is IDisposible
Maybe add using?

File: Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs [hunk 2]
```diff
@@ -40,43 +41,118 @@ public MapperManagementClient(IHttpClientFactory clientFactory, ILogger<MapperMa
...
+            }
+        }
+        
+        
+        private async Task<Result<Unit, ProblemDetails>> Post(string requestUri, HttpContent content, string languageCode = LocalizationHelper.DefaultLanguageCode, CancellationToken cancellationToken = default)
+        {
+            try
+            {
+                _httpClient.DefaultRequestHeaders.Add("Accept-Language", languageCode);
+                using var response = await _httpClient.PostAsync(requestUri, content, cancellationToken);
```
Comment (DFSko) [inline Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs:150]: HttpResponseMessage is IDisposible
Maybe add using?

File: Api/AdministratorServices/Models/Mapper/AccommodationData.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using HappyTravel.MapperContracts.Public.Accommodations.Enums;
+using HappyTravel.MapperContracts.Public.Accommodations.Internals;
+
+namespace HappyTravel.Edo.Api.AdministratorServices.Models.Mapper
+{
+    public readonly struct AccommodationData
+    {
+        public AccommodationData(string name, string category, ContactInfo contacts, LocationInfo location, List<ImageInfo> photos, AccommodationRatings rating, ScheduleInfo schedule, List<TextualDescription> textualDescriptions, PropertyTypes type, UniqueAccommodationCodes? uniqueCodes, string hotelChain, List<string> accommodationAmenities, Dictionary<string, string> additionalInfo)
```
Comment (DFSko) [inline Api/AdministratorServices/Models/Mapper/AccommodationData.cs:9]: string too loooooooong

File: Api/AdministratorServices/Models/Mapper/AccommodationSearchRequest.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using HappyTravel.MapperContracts.Public.Accommodations.Enums;
+using HappyTravel.SuppliersCatalog;
+
+namespace HappyTravel.Edo.Api.AdministratorServices.Models.Mapper
+{
+    public readonly struct AccommodationSearchRequest
+    {
+        public AccommodationSearchRequest(string nameQuery, int? countryId, int? localityId, int? localityZoneId, bool? isActive, List<AccommodationDeactivationReasons> deactivationReasons, string addressLineQuery, List<AccommodationRatings> ratings, List<Suppliers> suppliers, bool? hasDirectContract, int? skip, int? top)
```
Comment (DFSko) [inline Api/AdministratorServices/Models/Mapper/AccommodationSearchRequest.cs:9]: string too looooong

**Review by hasmikharutyunyan — APPROVED** [review]

File: Api/AdministratorServices/Models/Mapper/AccommodationSearchRequest.cs (hunk 4 above)
Comment (oleg-n-k) [reply Api/AdministratorServices/Models/Mapper/AccommodationSearchRequest.cs:9]: fixed

File: Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs (hunk 2 above)
Comment (oleg-n-k) [reply Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs:158]: 'using' directive is already there

File: Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs (hunk 1 above)
Comment (oleg-n-k) [reply Api/AdministratorServices/Mapper/AccommodationManagementServices/MapperManagementClient.cs]: fixed

File: Api/AdministratorServices/Models/Mapper/AccommodationData.cs (hunk 3 above)
Comment (oleg-n-k) [reply Api/AdministratorServices/Models/Mapper/AccommodationData.cs]: fixed

**Review by DFSko — APPROVED** [review]

**Review by AlexeyShok — APPROVED** [review]

**Pull request merged**